.github/workflows/ci.yaml

```diff
@@ -318,7 +318,7 @@ jobs:
       - name: Setup and execute Gradle 'integrationTestJdbc' task
         uses: gradle/gradle-build-action@v2
         with:
-          arguments: integrationTestJdbc -Dscalardb.jdbc.url=jdbc:sqlserver://localhost:1433 -Dscalardb.jdbc.username=sa -Dscalardb.jdbc.password=SqlServer19
+          arguments: integrationTestJdbc -Dscalardb.jdbc.url=jdbc:sqlserver://localhost:1433;encrypt=true;trustServerCertificate=true -Dscalardb.jdbc.username=sa -Dscalardb.jdbc.password=SqlServer19
```

Comment:
For the new SQL Server Driver version, we need these parameters according to the following article:
https://stackoverflow.com/questions/73291498/sql-server-spring-boot-pkix-path-building-failed-unable-to-find-valid-certific